components/code-input.json

```diff
@@ -1,5 +1,5 @@
 {
-  "codeInput": {
+  "code-input": {
```

Comment:
Can you change back to camelCase please (I want to get rid of all hyphens in keys and then I'll remove the code which handles it to ensure this is enforced)

Reply:
I changed this because I was already using `--code-input-whatever` and it stopped converting to kebab case. Do we really want `--codeInput-whatever` css vars?

Reply:
I think so, to distinguish between atomic names, and grouping. Could we compromise on codeinput-whatever?